src/Polar/polar.jl

```diff
@@ -24,7 +33,7 @@ giving a mathematical formulation with:
 
 ```
 """
-struct PolarForm{T, IT, VT, MT} <: AbstractFormulation where {T, IT, VT, MT}
+struct PolarForm{T, IT, VT, MT} <: AbstractPolarFormulation{T, IT, VT, MT}
```

Comment:
What difference does this change make?

Reply:
We have a new abstract type `AbstractPolarFormulation` that encapsulates both `PolarForm` and `BlockPolarForm`